framework/src/modules/nft/constants.ts

```diff
@@ -11,3 +11,31 @@
  *
  * Removal or modification of this copyright notice is prohibited.
  */
+
+export const LENGTH_CHAIN_ID = 4;
+export const LENGTH_NFT_ID = 16;
+export const LENGTH_COLLECTION_ID = 4;
+export const MIN_LENGTH_MODULE_NAME = 1;
+export const MAX_LENGTH_MODULE_NAME = 32;
+
+export const enum NftEventResult {
+	SUCCESSFUL = 0,
+	NFT_DOES_NOT_EXIST = 1,
+	NFT_NOT_NATIVE = 2,
+	NFT_NOT_SUPPORTED = 3,
+	NFT_LOCKED = 4,
+	NFT_NOT_LOCKED = 5,
```

Comment:
There is another module constant with same name `NFT_NOT_LOCKED` but is a string so I think we'd need to keep the event constant as `RESULT_NFT_NOT_LOCKED` same as what is in the LIP